File: src/main/java/rusha/x/Dovod.java
package rusha.x;

import java.util.Scanner;

public class Dovod {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        String b = new StringBuffer(String.valueOf(Math.abs(a))).reverse().toString();
        if (a >= 0)
            System.out.println(Integer.valueOf(b));
        else
            System.out.println(Integer.parseInt(b) * -1);
    }
}

File: src/main/kotlin/rusha/x/Dovod.kt
package rusha.x

import java.util.*
import kotlin.math.absoluteValue

fun main() {
    val scanner = Scanner(System.`in`)
    val a = scanner.nextInt()
    val b = a.absoluteValue.toString().reversed()
    if (a >= 0)
        println(b.toInt())
    else
        println(b.toInt() * -1)
}

